react-frontend/src/components/Charts.tsx

```diff
@@ -1,9 +1,52 @@
 import React from 'react';
+import { CategoryScale, Chart, Legend, LinearScale, LineElement, PointElement } from 'chart.js';
+import { Line } from 'react-chartjs-2';
 
-function Charts({ iterationCount }: { iterationCount: number }) {
-	const analyze = function () {};
+const chartOptions = {
+	responsive: true,
+	plugins: {
+		legend: {
+			position: 'top' as const
```

Comment:
I was curious about this line. Why is `as const` needed here but not for other strings?

Reply:
That's a good question. I copied and pasted this from the plugin author's documentation. If I take it away it is still happy. The [documented/debated reasoning makes sense](https://stackoverflow.com/questions/66993264/what-does-the-as-const-mean-in-typescript-and-what-is-its-use-case) but not really in this scenario.